pyproject.toml

```diff
@@ -38,6 +38,7 @@ pycairo = "^1.19"
 pangocffi = "^0.8.0"
 pangocairocffi = "^0.4.0"
 cairocffi = "^1.1.0"
+pyreadline = { version = "*", markers = "sys_platform == 'win32'" }
```

Comment:
update the lock file also. Run `poetry lock` for updating it.

Reply:
Wow that took a while to get working. But I have it now, so I can get to the last part of this pull request